hkmc2/shared/src/main/scala/hkmc2/utils/utils.scala

```diff
@@ -36,11 +37,16 @@ extension (t: Product)
       case Nil => "Nil"
       case xs: List[_] => "Ls of \n" + xs.iterator.map(aux(_)).mkString("\n").indent("  ")
       case s: String => s.escaped
-      case FldFlags(mut, spec, genGetter) =>
+      case TermDefFlags(mod) =>
+        val flags = Buffer.empty[String]
+        if mod then flags += "module"
```

Comment:
What's the use of this strange mutable buffer?

Reply:
IDK, I just copied it from below. I thought it was some good practice though 😪